### PR TITLE
DOCK-2557: Adds topic sentences in collection page

### PR DESCRIPTION
**Pull request by hyunnaye**
**Description**
This PR adds topic sentences to each entry in the collection page

Before:
image

After:
image

AI Bubble
image

**Review Instructions**
Go to https://qa.dockstore.org/organizations/BroadInstitute and verify that there are topic sentences for each entry.

**Issue**
https://ucsc-cgl.atlassian.net/browse/DOCK-2557

**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that your code compiles by running `npm run build`
- [x] Ensure that the PR targets the correct branch. Check the milestone or fix version of the ticket.
- [x] If this is the first time you're submitting a PR or even if you just need a refresher, consider reviewing our [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist)
- [x] Do not bypass Angular sanitization (bypassSecurityTrustHtml, etc.), or justify why you need to do so
- [x] If displaying markdown, use the `markdown-wrapper` component, which does extra sanitization
- [x] Do not use cookies, although this may change in the future
- [x] Run `npm audit` and ensure you are not introducing new vulnerabilities
- [x] Do due diligence on new 3rd party libraries, checking for CVEs
- [x] Don't allow user-uploaded images to be served from the Dockstore domain
- [x] If this PR is for a user-facing feature, create and link a documentation ticket for this feature (usually in the same milestone as the linked issue). Style points if you create a documentation PR directly and link that instead.
- [x] Check whether this PR disables tests. If it legitimately needs to disable a test, create a new ticket to re-enable it in a specific milestone.

### CODE REVIEW

File: src/app/organizations/collection/collection.component.html
Comment (kathy-t) [inline src/app/organizations/collection/collection.component.html:136]: Suggesting that we make the styling similar to the search page (smaller text, truncate to two lines with the full topic displayed in a tooltip).

We should also make sure we display the AI bubble if it's an unapproved AI topic

image

**Review by coverbeck — APPROVED** [review]
Approved, pending addressing of Kathy's comment.

File: src/app/organizations/collection/collection.component.html
Comment (coverbeck) [inline src/app/organizations/collection/collection.component.html:136]: On further reflection, shouldn't this be causing a build error until dockstore/dockstore#5980 is merged?

If so, the dependency might be worth clarifying in the PR description.

**Review by coverbeck — COMMENTED** [review]
Left an additional comment.

File: src/app/organizations/collection/collection.component.html
Comment (svonworl) [reply src/app/organizations/collection/collection.component.html:136]: > On further reflection, shouldn't this be causing a build error until [dockstore/dockstore#5980](https://github.com/dockstore/dockstore/pull/5980) is merged?
> 
> If so, the dependency might be worth clarifying in the PR description.

Would be great if there was a lightweight way to point the PR at a version of the webservice that supported the new `topic` field, so that reviewers could easily do things like check out the branch and test it.

**Review by svonworl — APPROVED** [review]

File: src/app/organizations/collection/collection.component.html
Comment (denis-yuen) [reply src/app/organizations/collection/collection.component.html:136]: Think we have part of this with https://github.com/dockstore/dockstore-ui2?tab=readme-ov-file#build pointing at branches or commits. I think the next part would be to upload artifacts to artifactory from branches other than develop

**Review by denis-yuen — APPROVED** [review]

**Review by kathy-t — APPROVED** [review]

**Pull request merged**
